src/objects/overview/overview.scss

```diff
@@ -0,0 +1,56 @@
+@use "../../compiled/tokens/scss/breakpoint";
+@use '../../mixins/ms';
+@use "../../compiled/tokens/scss/size";
+@use '../../mixins/fluid';
+
+/**
+ * The Overview stacks content vertically on small screens.
+ * On large screens the header and actions are stacked horizontally above the content.
+ * The specific spacing values were inferred from the Article Listing prototype.
+ */
+
+.o-overview__header {
+  margin-bottom: ms.step(1);
+}
+
+.o-overview__actions {
+  margin-bottom: ms.step(3);
+}
+
+@supports (display: grid) {
+  @media (min-width: breakpoint.$l) {
+    .o-overview {
+      @include fluid.column-gap(
+        breakpoint.$s,
+        breakpoint.$xl,
+        size.$spacing-gap-fluid-min,
+        size.$spacing-gap-fluid-max
+      );
```

Comment:
I'm torn on this. `column-gap` does seem to be [well supported in Grid](https://caniuse.com/mdn-css_properties_column-gap_grid_context), though not _quite_ as well supported as `grid-column-gap`. (We added this mixin for use with the `columns` property.)

I think it's probably fine, just typing out my initial pause.

Reply:
I had initially used `grid-column-gap` when building out this component but VS Code warned me it was deprecated and that I should use `column-gap`. The support for `column-gap` and `grid-column-gap` seems the same in Safari which is the main modern browser I'd worry about since updates are tied to OS updates.

That said, I don't have strong feelings about it and would be fine swapping over.